frontend/src/Components/ServiceProviderSignupStep1/ServiceProviderSignupStep1.jsx

```diff
@@ -48,18 +36,26 @@ const ServiceProviderSignupStep1 = ({ setStep }) => {
     }
     const phonePattern = /^\d{10}$|^\d{3}-\d{3}-\d{4}$/;
     if (!phonePattern.test(formData.phoneNumber)) {
-        setErrorMessage("Please enter a valid phone number.");
-        return;
+      setErrorMessage("Please enter a valid phone number.");
```

Comment:
Nice formatting cleanup : ) In the future we can keep all cleanup related code into it's own commit/PR, unless the touched lines directly relate to the new features being added